Gemfile.lock

```diff
@@ -1,37 +1,37 @@
 GEM
   remote: https://rubygems.org/
   specs:
-    CFPropertyList (2.2.8)
```

Comment:
It looks like you've run bundle update. Just because we don't know what the impact of this will be, we should test it separately.